src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -34,5 +34,8 @@
     @OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
     private User user;
```

Comment:
dodałbym poniższą adnotację lepiej to wygląda na bazie (hibernate tworzy user_user_id)

@JoinColumn(name = "user_id")